src/upgrade/upgrade_scripts/5.15.6/upgrade_bucket_policy.js

```diff
@@ -8,7 +8,11 @@ const _ = require('lodash');
 
 function _create_actions_map() {
     const actions_map = new Map();
-    for (const action of Object.values(OP_NAME_TO_ACTION)) {
+    // since we didn't have this API in version 5.14.0 and now the map includes array
+    // we delete this property not to harm this test
+    const op_name_to_action_without_get_object_attributes = { ...OP_NAME_TO_ACTION };
+    delete op_name_to_action_without_get_object_attributes.get_object_attributes;
+    for (const action of Object.values(op_name_to_action_without_get_object_attributes)) {
```

Comment:
Need to check if we plan to backport to 5.15, and then handle it instead of skipping
```suggestion
    for (const action of Object.values(OP_NAME_TO_ACTION)) {
        if (Array.isArray(action.regular)) {
            // API's with array actions were added after version 5.15.z (e.g get_object_attributes)
            // we skip this because there were no policies to fix
            continue;
        }
```

Reply:
Changed, thanks.